README.md

```diff
@@ -172,4 +174,13 @@ class Header extends React.Component {
 export default Header
 ```
 
-Now "hello egghead" will be rendered anywhere the old Header component was render.
+Now "hello egghead" will be rendered anywhere the old `Header` component was render.
+
+If you shadowing a component that references other theme components (and you still need them), You will need to import them. To import them, you start with `@eggheadio/gatsby-theme-egghead-blog/` and fill in the relative path of the component that you need.
```

Comment:
Added a part about importing addition theme components, the importing for those could be a bit confusing